puppet/zulip/manifests/hooks/send_zulip_update_announcements.pp

```diff
@@ -0,0 +1,9 @@
+# @summary Send zuip update announcements after deploy
```

Comment:
This hook isn't included automatically.  We need to add:
```
include zulip::hooks::send_zulip_update_announcements
```
...to `puppet/zulip/manifests/app_frontend_once.pp`